### ISSUE TITLE
"Expected committer but didn't get one"

### ISSUE THREAD

**Issue by docwhat** [termie/git-bzr-ng#14]
I did the following:

```
$ git-bzr clone bzr://bzr.mozilla.org/bugzilla/trunk bugzilla-bzr
Branched 7716 revision(s).ing revisions:Finishing stream:Done 79526/795269526
10:22:58 Calculating the revisions to include ...
10:22:58 Starting export of 7721 revisions ...
10:23:03 1000/7721 commits exported at 11281/minute 
10:23:14 2000/7721 commits exported at 7604/minute 
10:23:26 3000/7721 commits exported at 6496/minute 
10:23:38 4000/7721 commits exported at 6090/minute 
10:23:47 5000/7721 commits exported at 6111/minute 
10:23:54 6000/7721 commits exported at 6402/minute 
 fatal: Expected committer but didn't get one
 fast-import: dumping crash report to .git/fast_import_crash_12487
 bzr: broken pipe
 ERROR:root:bzr export failed
```

I don't know enough about bzr to know where the problem lies or how to fix it. :-/

**Comment by termie** [comment]
great, thanks for the bug report, i'll check this out

**Comment by justinsb** [comment]
I had the same issue when I was using the bzr that came with Ubuntu M (2.2.1)

When I uninstalled and installed using pip install, I wasn't able to install 2.2 (not found), but I was able to just "pip install bzr".  I got bzr 2.3.1.  It got me past the "expected committer but didn't get one".  I am having trouble pushing but haven't diagnosed that yet.

(I thought I already posted this; apologies if this is duplicated)

**Comment by termie** [comment]
pip install bzr==2.2 failed? :( grr, it looks like they pushed 2.3.0 (instead of the beta versions) but i suspect they maintained the bug that affects pushing

**Comment by justinsb** [comment]
It definitely "fails for me":

$ sudo pip install bzr==2.2
Downloading/unpacking bzr==2.2
  Could not find a version that satisfies the requirement bzr==2.2 (from versions: )
No distributions matching the version for bzr==2.2
Storing complete log in /home/justinsb/.pip/pip.log

Is the pushing a known bug?  Should I be installing bzr from 2.2 source manually then?

**Comment by termie** [comment]
Yeha, it's a known bug: https://bugs.launchpad.net/bzr/+bug/541626

this is disappointing :/ I'd suggest installing 2.2 from source, there is a simple patch to fix the issue but it seems more trouble than installing 2.2 from source.

**Comment by justinsb** [comment]
I believe I have resolved the push issue that I was experiencing - I wasn't getting the "BTreeBuilder" message, I was getting a message about an extra argument.  I traced that down to bzr-fastimport trunk.  See issue #19 for a "works for me" resolution :-)

**Comment by termie** [comment]
this appears to be fixed once upgrading to bzr 2.3.0
